tests/microos/workloads/x11-container/x11_helm_server.pm

```diff
@@ -0,0 +1,86 @@
+# SUSE"s openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Deploy X11, pulseaudio, firefox kiosk
+#          with Kubernetes Using Helm
+# Maintainer: Grace Wang <grace.wang@suse.com>
+
+use base 'consoletest';
+use warnings;
+use strict;
+use testapi;
+use lockapi;
+use mmapi;
+use utils;
+use mm_network 'setup_static_mm_network';
+use containers::k8s;
+use transactional;
+
+# MM network check: try to ping the gateway, the client and the internet
+sub ensure_client_reachable {
+    assert_script_run('ping -c 1 10.0.2.2');
+    assert_script_run('ping -c 1 10.0.2.102');
+    assert_script_run('curl conncheck.opensuse.org');
+}
+
+sub run {
+    my ($self) = @_;
+    my $release_name = "kiosk";
+    my $namespace = "kiosk";
+    my $helm_values = "kiosk_values.yaml";
+    my $helm_chart = "oci://registry.suse.de/suse/sle-15-sp6/update/bci/helmcharts/suse/kiosk/kiosk-chart";
```

Comment:
```suggestion
    my $helm_chart = get_required_var("HELM_CHART");
```
Because there is a version dependency, I think it's better to make this an openQA setting.